Jenkinsfile-Agent-Test-2

```diff
@@ -82,12 +82,6 @@ pipeline {
             parallel {
                 stage('Group1') {
                     stages {
-                        stage('spring-tx 4.x+ (10)') {
-                            steps {
-                                sh 'bash test/plugin/run.sh spring-tx-scenario'
-                            }
-                        }
-
```

Comment:
Hi, Notice that the ```spring-tx``` is deleted here.

Reply:
I don't know why. May be i lost. I've restore.